deploy.sh

```diff
@@ -65,6 +67,16 @@ while (( "$#" )); do
             BUILD_IMAGE="--build"
             shift
             ;;
+        -R|--replicas)
+            if [[ "$2" =~ ^[0-9]+$ ]]; then
```

Comment:
I have previously stated I think bash is not appropriate for writing system scripts PARTICULARLY ones that can/will be run as root.  I believe (and I've probably been writing shell scripts as long as you've been on two feet: as a rebellious youth I once started a system backup script in the (then) default interactive shell (csh) only to find out why the older, crankier guys said it shouldn't be done).  Now I'm the old, cranky guy, and I beleieve that *ONLY* POSIX "sh" should be used.  See https://en.wikipedia.org/wiki/Shellshock_(software_bug) and convince me nothing like this could happen again.    In addition, I think validating that the given argument is numeric is excessive, and only adds bloat:  If this script needs to be "fool proofed" then something has gone wrong in the hiring process: the story-indexer deploy.sh script runs `stack config -c docker-compose.yml` to validate that the generated YML file is sane.